editors/code/src/installation/download_file.ts

```diff
@@ -14,31 +19,34 @@ export async function downloadFile(
     destFilePermissions: number,
     onProgress: (readBytes: number, totalBytes: number) => void
 ): Promise<void> {
-    const res = await fetch(url);
+    const res = await fetch(url).catch(NestedError.rethrow("Failed at initial fetch"));
```

Comment:
I still don't think that using non-ubiquitous helper libraries is a good tradeoff.

Reply:
Sorry, didn't understand you

Reply:
Will remove NestedError then...